Source/Config/Main.lua

```diff
@@ -0,0 +1,50 @@
+Auctionator.Config.Option = {
+  DEBUG = "debug",
+  MAILBOX_TOOLTIPS = "mailbox_tooltips",
+  VENDOR_TOOLTIPS = "vendor_tooltips",
+  AUCTION_TOOLTIPS = "auction_tooltips",
+  ENCHANT_TOOLTIPS = "enchant_tooltips",
+  SHOW_LISTS = "show_lists",
+}
+
+local defaults = {
+  [Auctionator.Config.Option.DEBUG] = false,
+  [Auctionator.Config.Option.MAILBOX_TOOLTIPS] = true,
+  [Auctionator.Config.Option.VENDOR_TOOLTIPS] = true,
+  [Auctionator.Config.Option.AUCTION_TOOLTIPS] = true,
+  [Auctionator.Config.Option.ENCHANT_TOOLTIPS] = true,
+  [Auctionator.Config.Option.SHOW_LISTS] = true,
+}
+
+function validOption(name)
```

Comment:
Nitpick: I'd name this `IsValidOption` so that the intent of the function is clearer.

I think we should also make this function local if we're not namespacing it.